### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    name: Continuous integration
+    runs-on: ubuntu-18.04
+    timeout-minutes: 60
+    env:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      TX_USR: ${{ secrets.TX_USR }}
+      TX_PWD: ${{ secrets.TX_PWD }}
+      TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+      TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```
Comment (sbrunner) [inline .github/workflows/ci.yaml:19]: Why not using secrets in gopass by using `camptocamp/initialise-gopass-summon-action`?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        run: sudo apt-get install -y gettext python3-virtualenv
+      - uses: actions/checkout@v1
+      - name: Build
+        run: make build
+      - name: Lint
+        run: make check
+      - name: Tests
+        run: make test
+      - name: Docker compose logs
+        if: ${{ always() }}
```
Comment (sbrunner) [inline .github/workflows/ci.yaml:31]: `{{` are not needed ...
```suggestion
        if: always()
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      - name: Deploy PyPI
+        if: >
+          github.repository == 'camptocamp/c2cgeoform'
+          && success()
+          && (
+            startsWith(github.ref, 'refs/tags/')
+          )
+        run: |
+          sudo python3 -m pip install twine wheel
+          scripts/deploy-pypi
```
Comment (sbrunner) [inline .github/workflows/ci.yaml:44]: Why not using c2cciutils to deploy?

**Review by sbrunner — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (arnaud-morvan) [reply .github/workflows/ci.yaml:17]: I've open a dedicated issue: https://github.com/camptocamp/c2cgeoform/issues/239

File: .github/workflows/ci.yaml (hunk 3 above)
Comment (arnaud-morvan) [reply .github/workflows/ci.yaml:45]: I've open a dedicated issue: https://github.com/camptocamp/c2cgeoform/issues/240

**Pull request merged**